gehomesdk/erd/erd_configuration.py

```diff
@@ -203,4 +203,9 @@ def erd_encode(self, value: Any) -> str:
     ErdConfigurationEntry(ErdCode.SAC_SLEEP_MODE, ErdOnOffConverter(), ErdCodeClass.AC_SENSOR),
     ErdConfigurationEntry(ErdCode.SAC_TARGET_TEMPERATURE_RANGE, ErdSacTargetTemperatureRangeConverter(), ErdCodeClass.AC_SENSOR),
     ErdConfigurationEntry(ErdCode.SAC_AUTO_SWING_MODE, ErdOnOffConverter(), ErdCodeClass.AC_SENSOR),
+
+    #Hood
+    ErdConfigurationEntry(ErdCode.HOOD_LIGHT_MODE, ErdHoodLightSettingConverter(), ErdCodeClass.HOOD_SENSOR),
+    ErdConfigurationEntry(ErdCode.HOOD_FAN_MODE, ErdHoodFanSettingConverter(), ErdCodeClass.HOOD_SENSOR),
```

Comment:
We have a couple "fan" controls now, perhaps we should make a fan code class here (i.e. it'll show a fan icon in HA)?

Reply:
It could... There are a few things like this - timer is another one as well. My hood is also using an ERD labeled "OVEN_TIMER"... I'm not worried yet about this - opportunity as this evolves is what I'm thinking.

Reply:
Yeah, there's already a timer code class.  It's a bit of a pain to add some of this after (requires a new published version of the library before it's available for HA), that's why I'm thinking we might want to do it now so we can one-shot the SDK updates.

What other ERDs are showing up for the hood?

Reply:
I'm in no rush to publish or even merge any of this. I just exec into the container and 'install the version' I have locally to run and it works fine while I'm tinkering. So none of this needs to be merge for the time being to continue on what I'm looking at with the next component.

It has timer capability, it also has the ability to schedule the lights to turn on/off which I have no plans on supporting. The entire point of integration is to use a system like HA to do those types of things so I didn't see the value. I'm guessing some of the codes are related to that.

Here's the codes:
```
2021-08-18 16:59:46,334 DEBUG    Setting ErdCode.MODEL_NUMBER to UVW9301SL1SS
2021-08-18 16:59:46,334 DEBUG    Setting ErdCode.SERIAL_NUMBER to
2021-08-18 16:59:46,334 DEBUG    Setting ErdCode.APPLIANCE_TYPE to ErdApplianceType.HOOD
2021-08-18 16:59:46,334 DEBUG    Setting ErdCode.UNKNOWN_0099 to b'\x00'
2021-08-18 16:59:46,334 DEBUG    Setting ErdCode.WIFI_MODULE_SW_VERSION to 0.0.5.97
2021-08-18 16:59:46,334 DEBUG    Setting ErdCode.WIFI_MODULE_SW_VERSION_AVAILABLE to 0.0.0.0
2021-08-18 16:59:46,334 DEBUG    Setting ErdCode.ACM_UPDATING to False
2021-08-18 16:59:46,334 DEBUG    Setting ErdCode.APPLIANCE_SW_VERSION to 0.0.0.0
2021-08-18 16:59:46,335 DEBUG    Setting ErdCode.APPLIANCE_SW_VERSION_AVAILABLE to 0.0.0.0
2021-08-18 16:59:46,335 DEBUG    Setting ErdCode.APPLIANCE_UPDATING to False
2021-08-18 16:59:46,335 DEBUG    Setting 0x500F to b'\x01'
2021-08-18 16:59:46,335 DEBUG    Setting 0x5020 to b'\x00\x00\x00\x00\x00\x00'
2021-08-18 16:59:46,335 DEBUG    Setting ErdCode.UPPER_OVEN_KITCHEN_TIMER to 0:00:00
2021-08-18 16:59:46,335 DEBUG    Setting ErdCode.HOOD_FAN_MODE to ErdHoodFanSetting.OFF
2021-08-18 16:59:46,335 DEBUG    Setting 0x5B01 to b'\x1f'
2021-08-18 16:59:46,335 DEBUG    Setting ErdCode.HOOD_LIGHT_MODE to ErdHoodLightSetting.OFF
2021-08-18 16:59:46,335 DEBUG    Setting 0x5B03 to b'\x07'
2021-08-18 16:59:46,335 DEBUG    Setting 0x5B04 to b'\x00'
2021-08-18 16:59:46,335 DEBUG    Got initial appliance type for GeAppliance(D828C92C0F11) (ErdApplianceType.HOOD)
```

Reply:
0x5b04 = DELAY (not sure what the context for a delay is for a hood?)
0x5b03 = LIGHT_LEVEL_AVAILABILITY (flag, 1 = low available, 2 = mid available, 3 = high available)
0x5b01 = FAN_SPEED_AVAILBILITY (flag, 1, 2, 4, 8, 16)
0x5020 = TIMER (in seconds?)
0x500F = TIMER_AVAILABILITY (ErdOnOff)

Let me know if you want me to implement these, or if you want to do it.  We'll need these so that we can create sensors appropriately for multiple models in HA (based on these, not every hood has all the features).

All scheduling-type functions for SmartHQ are actually in the app, so none of that is implemented for other appliances either (agree that that's the whole point of exposing to HA).

Reply:
actually that would be great! it appears then there is some bit-encoding for fan speed and such. If you already understand this then that would be helpful to add those for me.

Reply:
I'll take care of that tomorrow then, and get all these ERDs defined after merging in your changes.  Thanks for your help!

Reply:
No problem! Thank you for your work on this, its worked well so far (fridge and laundry). It will be great to have support for all of these appliances in HA.